recipes/bob.extension/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON -B setup.py install --single-version-externally-managed --record record.txt
```

Comment:
> This might make sense for this specific package but since these are automated pulls and starting from the next one they will have custom code in `build.sh`

Sorry. I don't get it. What are the automated pulls  you are talking about? Also. if `bob.<another>` needs a more elaborated `build.sh` you can do it even though this one will still be inside the build script. We prefer less files and compactness to consistency. Otherwise we would have `build.sh` for all the packages for conda-forge consistency.

With that said. I don't have any strong feelings about it. If keeping the `build.sh` is important to you that is fine.

@jakirkham back to you. Feel free to merge once you are OK.